src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Triggers/ServiceBusTriggerBindingStrategy.cs

```diff
@@ -125,7 +125,7 @@ internal static void AddBindingData(Dictionary<string, object> bindingData, Mess
                 deliveryCounts[i] = messages[i].SystemProperties.DeliveryCount;
                 deadLetterSources[i] = messages[i].SystemProperties.DeadLetterSource;
                 lockTokens[i] = messages[i].SystemProperties.IsLockTokenSet ? messages[i].SystemProperties.LockToken : string.Empty;
-                expiresAtUtcs[i] = messages[i].ExpiresAtUtc;
+                expiresAtUtcs[i] = messages[i].ExpiresAtUtc.ToUniversalTime();
```

Comment:
Please leave a comment indicating this is temporary until the Service Bus SDK addresses the missing timezone issue.  Same for the other area in this file.

Reply:
Done